### PR TITLE
🎉 Hubspot: Added deals associations for company and contacts

### PR DESCRIPTION
**Pull request by stanleyivan96**
## What
Added companies and contact associations to hubspot deals
#2983 

## How
1. Added companies and contact associations to client.py under deals
2. Added companies and contact associations to the schema deals.json

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py`
2. `airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/deals.json`
3. the rest

### CODE REVIEW

**Review by sherifnada — APPROVED** [review]
Thanks @stanleyivan96 ! We'll get this merged soon.

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py [hunk 1]
```diff
@@ -81,8 +81,10 @@ def streams(self) -> Iterator[AirbyteStream]:
         for stream in super().streams:
             properties = self._apis[stream.name].properties
             if properties:
-                stream.json_schema["properties"]["properties"] = {"type": "object", "properties": properties}
-            stream.default_cursor_field = [self._apis[stream.name].updated_at_field]
+                stream.json_schema["properties"]["properties"] = {
+                    "type": "object", "properties": properties}
+            stream.default_cursor_field = [
+                self._apis[stream.name].updated_at_field]
```
Comment (htrueman) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py:87]: This should be formatted like that. Please check out the `./gradlew format` command https://docs.airbyte.io/contributing-to-airbyte/developing-locally#run-formatting-automation-tests
```suggestion
                stream.json_schema["properties"]["properties"] = {"type": "object", "properties": properties}
                stream.default_cursor_field = [self._apis[stream.name].updated_at_field]
```

**Comment by marcosmarxm** [comment]
@htrueman i think Davin is working to run the CI from a fork. I tried yesterday and didn't succeed too

**Comment by htrueman** [comment]
Merged the updated copy of this PR [here](https://github.com/airbytehq/airbyte/pull/3850).
Also check out the related [issue](https://github.com/airbytehq/airbyte/issues/3927).

Closing this PR.

**Pull request closed without merging**